app/view/templates/layout.php

```diff
@@ -23,37 +25,31 @@
     <?php endif ?>
 
     <?php if (isreportingerrors()) : ?>
-    <script>
-        const sentrydsn = '<?= Wcms\Config::sentrydsn() ?>';
-        const version = '<?= getversion() ?>';
-        const url = '<?= Wcms\Config::url() ?>';
-        const basepath = '<?= Wcms\Config::basepath() ?>';
-    </script>
-    <script type="module" src="<?= Wcms\Model::jspath() ?>sentry.bundle.js"></script>
+        <script>
+            const sentrydsn = '<?= Wcms\Config::sentrydsn() ?>';
+            const version = '<?= getversion() ?>';
+            const url = '<?= Wcms\Config::url() ?>';
+            const basepath = '<?= Wcms\Config::basepath() ?>';
+        </script>
+        <script type="module" src="<?= Wcms\Model::jspath() ?>sentry.bundle.js"></script>
     <?php endif ?>
 </head>
-
-
-
-<?php
-if (!empty($flashmessages) && is_array($flashmessages)) : ?>
-<a href="#flashmessage">
-    <div class="flashmessage" id="flashmessage">
-        <ul>
-            <?php foreach ($flashmessages as $flashmessage ) : ?>
-                <li class="alert alert-<?= $flashmessage['type'] ?>">
-                    <?= $flashmessage['content'] ?>
-                </li>
-            <?php endforeach ?>
-            </ul>
-        </div>
-    </a>
-<?php endif ?>
-
 <body>
```

Comment:
Oh! interesting, I totally forgot that I've put flash messages out of body. Seems better like this !